generated/components.json

```diff
@@ -4652,6 +4614,59 @@
         }
       ]
     },
+    "tooltip": {
+      "id": "tooltip",
+      "name": "Tooltip",
+      "status": "alpha",
+      "a11yReviewed": false,
+      "stories": [
+        {
+          "id": "components-tooltip--default",
+          "code": "() => (\n  <Box\n    sx={{\n      p: 5,\n    }}\n  >\n    <Tooltip aria-label=\"Filter vegetarian options\" direction=\"e\">\n      <Button>🥦</Button>\n    </Tooltip>\n  </Box>\n)"
+        }
+      ],
+      "props": [
+        {
+          "name": "align",
+          "type": "'left' | 'right'"
+        },
+        {
+          "name": "aria-label",
+          "type": "string",
+          "description": "Should be utilised for label type tooltips and it is going to be used to label the tooltip trigger."
+        },
+        {
+          "name": "direction",
```

Comment:
What do you think about adding the default value `"defaultValue": "n"` to the component documentation to make the default direction explicit?